plugins/catalog/api-report.md

```diff
@@ -184,8 +185,10 @@ export const CatalogTable: {
 export interface CatalogTableProps {
   // (undocumented)
   actions?: TableProps<CatalogTableRow>['actions'];
+  // Warning: (ae-forgotten-export) The symbol "CatalogTableColumnsFunc" needs to be exported by the entry point index.d.ts
```

Comment:
Ah, could you fix this one please? 🙏 It just needs to be exported alongside the rest

Reply:
Oops

Reply:
I also moved it into `types.ts` which felt like a better home for it